build/helper/metadata_add_all.py

```diff
@@ -77,13 +73,27 @@ def _add_is_error_handling(f):
 
 def _add_buffer_info(parameter):
     '''Adds buffer information to the parameter metadata iff 'size' is defined else assume not a buffer'''
+
+    # We are going to treat ViChar[], ViString, ViConstString and ViRsrc the same. For simplicity in the generated
```

Comment:
reword comment for clarity:

For simplicity, we are going to treat ViChar[], ViString, ViConstString, and ViRsrc the same: As ViChar and is_buffer True

Reply:
Fixed